src/utils/index.ts

```diff
@@ -15,21 +15,21 @@
 	along with this program.  If not, see <https://www.gnu.org/licenses/>.
 */
 
-import {
-	PinejsClientCoreFactory,
-	PinejsClientRequest,
-} from 'pinejs-client-request';
-export { PinejsClientCoreFactory } from 'pinejs-client-request';
-import * as pkg from 'pjson';
 import * as winston from 'winston';
 
-export type AnyObject = PinejsClientCoreFactory.AnyObject;
+import { PinejsClientRequest } from 'pinejs-client-request';
+export { PinejsClientCoreFactory } from 'pinejs-client-request';
+
+import { version as VERSION } from '../../package.json';
+export { VERSION };
+
+import * as errors from './errors';
+export { errors };
```

Comment:
I think it'd make sense for other modules to import `utils/error` instead, I don't see a reason to alias internal modules (the main one I'm aware of is for the case of exposing a single entrypoint for a node module, but that doesn't apply here)